src/main/java/com/hazelcast/aws/utility/RetryUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.aws.utility;
+
+import com.hazelcast.core.HazelcastException;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.util.ExceptionUtil;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Static utility class to retry operations related to connecting to AWS Services.
+ */
+public final class RetryUtils {
+    static final long INITIAL_BACKOFF_MS = 1500L;
+    static final long MAX_BACKOFF_MS = 5 * 60 * 1000L;
+    static final double BACKOFF_MULTIPLIER = 1.5;
```

Comment:
Pretty reasonable values: Maybe in the future we could make them configurable (nice to have but not needed for this PR!)